Quoridor.Model/Position.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Quoridor.Model
+{
+    class Position
+    {
+        public int x { get; }
```

Comment:
Return there constructor (don't delete fields) and You can merge it or wait for second review of @Anastasia-Tre

Reply:
But You will need to reassign Her as reviewer, if You prefer second variant

Reply:
I will wait for her